### ISSUE TITLE
Make website more fun for beginners to improve

### ISSUE THREAD

**Issue by treyhunner** [pythonsd/pythonsd.org#70]
It might be fun for beginners to improve the SD Python website.

Some ideas:

- Use Django/Pyramid/Flask instead of pelican
- Host on Heroku or somewhere else that's easy to deploy to
- Add setup instructions and add instructions to the website to let beginners know they can improve it
- Add an AUTHORS file (yay credit!)
- Encourage attendees of the Saturday study group to work on the website

**Comment by willingc** [comment]
Very +1 to this :)  :penguin: OpenHatch all the things ;)

**Comment by davidfischer** [comment]
I'm +1 to it.

**Comment by macro1** [comment]
Huh. I didn't even see this. @treyhunner mentioned wanting something more beginner friendly and familiar to experienced Python devs as well, so I started a Django project with that in mind.

**Comment by treyhunner** [comment]
@macro1 where's the Django project you started?  Where should we host it?

**Comment by alaindomissy** [comment]
https://github.com/pythonsd/pythonsd-django  ?

On Sat, Aug 1, 2015 at 11:25 PM, Trey Hunner <notifications@github.com>
wrote:

> @macro1 <https://github.com/macro1> where's the Django project you
> started? Where should we host it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonsd/pythonsd.org/issues/70#issuecomment-126994568>
> .
>



-- 
--Alain
619-884-5450

**Comment by macro1** [comment]
Yeah, that's the project. We can put it on Heroku for now while it's under development unless you have a better idea.
